recipes/tethys-platform/bld.bat

```diff
@@ -0,0 +1,2 @@
+set PBR_VERSION=3.4.3
```

Comment:
```suggestion
set PBR_VERSION=$PKG_VERSION
```
https://docs.conda.io/projects/conda-build/en/latest/user-guide/environment-variables.html#environment-variables-set-during-the-build-process

Reply:
Done.